joint_velocity_example_controller:
    type: panda_mpc/JointVelocityExampleController
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7


panda_controller:
    type: panda_mpc/JointVelocityExampleController
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7

panda_mpc:
    type: panda_mpc/PandaMPCController
    arm_id: panda
    joint_names:
        - panda_joint1
        - panda_joint2
        - panda_joint3
        - panda_joint4
        - panda_joint5
        - panda_joint6
        - panda_joint7
    control_level : velocity
    p_gains_: [10, 10, 10, 10, 10, 10]
    d_gains_: [0.0 ,0.0, 0.0, 0.0, 0.0, 0.0]
    p_gains_qd_: [5, 5, 5, 5, 5, 5, 5]
    root_link_ : "panda_link0"
    tip_link_ : "panda_link8"
    q_mean_ : [0.01 , 0.0, 0.01, -1.501, 0.0, 1.5, 0.0]
    qd_min_ : [-1.0 , -1.0 , -1.0 , -1.0 , -1.0 , -1.0 , -1.0]
    qd_max_ : [1.0 , 1.0 , 1.0 , 1.0 , 1.0 , 1.0 , 1.0]
    qdd_max_ : [10 , 7.5 , 8 , 7.5 , 10 , 10 , 10]
    torque_max_ : [67.0 , 67.0 , 67.0 , 67.0 , 12.0 , 12.0 , 12.0]
    init_pos_attend_ : true
    regularisation_weight_ : 1.0e-5
    N_ : 3
    dt_ : 0.05
    dsafe_ : 0.1
    robot_member_ : 1
    robot_vertices_ : 1
    obstacle_member_: 1
    obstacle_vertices_ : 1
